src/vm/moar/dispatchers.nqp

```diff
@@ -3929,7 +3929,8 @@ nqp::dispatch('boot-syscall', 'dispatcher-register', 'raku-isinvokable', -> $cap
             my sub runtime-only($t) {
                 return 0 if nqp::isnull($t);
                 nqp::isconcrete(nqp::how($t).refinement($t)) 
-                    || (nqp::how($t).archetypes.nominalizable && runtime-only(nqp::how($t).wrappee-lookup($t, :subset)))
+                    || nqp::how(my $refinee := nqp::how($t).refinee($t)).archetypes.nominalizable
```

Comment:
I think this would be easier to read if you spell it `(my $refinee := $t.HOW.refinee($t)).HOW.archetypes.nominalizable`

Reply:
BTW, it makes sense to use `nqp::how_nd` here.